supabase-openai-summary/README.md

```diff
@@ -0,0 +1,150 @@
+# Generate Daily Summary of Supabase Database using GPT-4 and ZenML
+
+This project demonstrates how to create a daily summary of a [Supabase](https://supabase.com) database using [OpenAI GPT-4](https://openai.com/gpt4) and [ZenML](https://zenml.io). We use the YouTube video titles from [you-tldr](https://you-tldr.com) as an example and generate a summary of the last 24 hours of visitor activity. ZenML versions all data, allowing GPT-4 to compare the current summary to the previous one. The pipeline is executed daily using GitHub Actions and a [a deployed ZenML instance in Hugging Face Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker-zenml).
+
+The output of the pipeline posts the summary to a Slack channel:
+
+![The summary in Slack](assets/youtldr_summarizer_slack.png)
+
+An example report might be:
+
+1. There is a significant interest in technology and software development, with topics such as data analytics, programming languages, software tools, and game development being popular among users.
+
+2. Users are also interested in educational and informative content, including tutorials, lectures, and discussions on various subjects such as ethics, philosophy, and economics.
+
+3. There is a noticeable trend in content related to personal development and self-improvement, with videos on productivity, decision-making, and life advice.
```

Comment:
This seems redundant since an example is already shown in the PNG right above.

Reply:
I thought if you skipped the image it might make snese to read it

Reply:
Yeah both fine with me 👍